src/com/t_oster/visicut/VisicutModel.java

```diff
@@ -757,6 +757,24 @@ public void sendJob(String name, ProgressListener pl, Map<LaserProfile, List<Las
     }
   }
 
+  public void saveJob( String name, ProgressListener pl, Map<LaserProfile, List<LaserProperty>> props, List<String> warnings ) throws IllegalJobException, SocketTimeoutException, Exception{
+	  LaserCutter lasercutter = this.getSelectedLaserDevice().getLaserCutter();
```

Comment:
As the counterpart in LibLaserCut, this method should get an output stream where to put the data.